src/endoplasmic_reticulum.cpp

```diff
@@ -0,0 +1,25 @@
+#include <morphio/properties.h>
+#include <morphio/endoplasmic_reticulum.h>
+
+namespace morphio {
+const std::vector<uint32_t>& EndoplasmicReticulum::sectionIndices() const
+{
+    return _properties->_endoplasmicReticulumLevel._sectionIndex;
+}
+
+const std::vector<float>& EndoplasmicReticulum::volumes() const
```

Comment:
how is the lifetime of the vector guaranteed?

Reply:
The lifetime will be the one of the Morphology object.